backend/src/test/java/ca/bc/gov/hlth/mohums/integration/MoHUmsIntegrationTests.java

```diff
@@ -200,6 +200,17 @@ public void addUserGroups() throws Exception {
                 .expectStatus().isEqualTo(HttpStatus.NO_CONTENT); //204 indicates success
     }
 
+    @Test
+    public void removeUserGroups() throws Exception {
+        webTestClient
+                .delete()
+                // 123-tschiavo user
+                .uri("users/39f73cbd-dbf0-41c6-a45c-997c44c1c952/groups/1798203d-027f-4856-a445-8a90c1dc9756")
+                .header("Authorization", "Bearer " + jwt)
+                .exchange()
+                .expectStatus().isEqualTo(HttpStatus.NO_CONTENT); //204 indicates success
+    }
```

Comment:
If we're deleting this user, how will it be recreated for other tests? Or are we not bothering.

Reply:
we aren't deleting the user just the group association, the comment could be made better by also including the group name. 
It still gives a 204 regardless of if the group is there to remove or not